engine/schema/src/main/resources/META-INF/db/schema-41710to41800.sql

```diff
@@ -22,4 +22,11 @@
 -- Enable CPU cap for default system offerings;
 UPDATE `cloud`.`service_offering` so
 SET so.limit_cpu_use = 1
-WHERE so.default_use = 1 AND so.vm_type IN ('domainrouter', 'secondarystoragevm', 'consoleproxy', 'internalloadbalancervm', 'elasticloadbalancervm');
\ No newline at end of file
+WHERE so.default_use = 1 AND so.vm_type IN ('domainrouter', 'secondarystoragevm', 'consoleproxy', 'internalloadbalancervm', 'elasticloadbalancervm');
+
+-- Drop all backup records and change date column type to DATETIME. The data will be resynchronized automatically later;
+DELETE FROM `cloud`.`backups`
```

Comment:
Wouldn't that cause data loss for existing backup entries? Why not modify the column date/time?

Reply:
@JoaoJandre I see your point that even if these backups are deleted from the database, the management server will again sync and get all the backups from Veeam. But instead of losing the history of backups, it is better to modify the date. You can use java upgrade file Upgrade417100to418000.java file to do that.

By deleting the entries we will also lose the history of removed backups.